xml/art_sle_ha_pmremote.xml

```diff
@@ -459,28 +457,33 @@ Full list of resources:
    </procedure>
   </sect2>
   <sect2 xml:id="sec.ha.pmremote.config.uc1.start.resources">
-   <title>Starting Resources on Remote Node</title>
+   <title>Starting Resources on the Remote Node</title>
    <para>
-    Once the remote node is integrated into the cluster, starting resources
-    on a remote node is the exact same as on cluster nodes.
+    After the remote node is integrated into the cluster, you can start
+    resources on the remote node in the same way as on any cluster node.
    </para>
    <warning>
+    <title>Restrictions Regarding Groups and Constraints</title>
     <para>
      Never involve a remote node connection resource in a resource group,
      colocation constraint, or order constraint.
     </para>
+    <para>
+     <remark>taroth 2017-08-09: why? would be good to give an explanation
```

Comment:
Good question. It because this resource is so *special*.  Not like all the other resources , this resource is actually handled internally in crmd daemon. It establishes and manages the the connection
between a cluster node  and  the remote node. It basically defines the existence of the remote node.  So practically, it doesn't  make sense to define order/colocation constraints between a "remote node" and a resource,   and such constraints might introduce issues when cluster handles transitions (-- Instead, we can use location constraints to define the relationships between the remote node and resources as usual if necessary).

I'm not sure if my explanation is making things even more confusing :-)  If so, we could omit the details and simply leave just the warning.